rest/data/cli_update.go

```diff
@@ -21,23 +21,26 @@ func (c *CLIUpdateConnector) GetCLIUpdate() (*model.APICLIUpdate, error) {
 		}
 	}
 
-	if err := update.BuildFromService(*config); err != nil {
+	flags, err := evergreen.GetServiceFlags()
+	if err != nil {
 		return nil, gimlet.ErrorResponse{
 			StatusCode: http.StatusInternalServerError,
 			Message:    err.Error(),
 		}
 	}
+	if flags.S3BinaryDownloadsDisabled {
```

Comment:
I think setting the field to nil in the API model afterwards is more future-proof but it doesn't really matter

Reply:
I think it's probably okay since the API fields map 1:1 with the ClientConfig fields.